web/themes/custom/novel/templates/layout/page--user--registration--create.html.twig

```diff
@@ -0,0 +1,52 @@
+{#
+/**
+ * @file
+ * Theme override to display a user/registration/create page.
```

Comment:
Great that you write that the doctype, html, head and body tags have been left out of this template.
But I think also it makes sense to write why.

Reply:
Actually this is not something I have added. It originates from the default page.html.twig file. 

But maybe I should instead change the description and say that this template is a variant of the page.html.twig template, but without the header and footer?